planet/scripts/v1.py

```diff
@@ -247,6 +250,23 @@ def series():
     pass
 
 
+@series.command('time-series')
```

Comment:
Would it make more sense to have this be `planet mosaics series download` instead of `planet mosaics series time-series`?